content/en/security/cloud_siem/entities_and_risk_scoring.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Cloud SIEM Risk Based Entity Insights
+---
+
+{{< callout url=""https://datadoghq.com/private-beta/risk-based-entity-insights" header="Join the Beta!">}}
+Risk Based Entity Insights is in private beta.
+{{< /callout >}}
+
+## Overview
+
+[Cloud SIEM's Risk Based Entity Insights][4] consolidates multiple data sources, such as SIEM threats and CSM insights, into a profile representing a single security entity, such as an IAM user.
+
+With Risk Based Entity Insights, you can:
+
+- Explore entities, filtering them by attributes such as [risk score](#risk-scoring) or entity type.
+- View all data relevant to an entity, such as signals, misconfigurations, and identity risks.
+- Triage relevant items in bulk.
+- Take mitigation steps such as creating an automated workflow or creating a global case for an entity.
```

Comment:
Take mitigation steps such as creating **a global suppression** or creating a case for an entity.